pkg/minikube/constants/constants.go

```diff
@@ -39,8 +39,21 @@ const (
 	ClusterDNSDomain = "cluster.local"
 	// DefaultServiceCIDR is The CIDR to be used for service cluster IPs
 	DefaultServiceCIDR = "10.96.0.0/12"
+	// DockerTLSVerifyEnv is used for docker daemon settings
+	DockerTLSVerifyEnv = "DOCKER_TLS_VERIFY"
+	// DockerHostEnv is used for docker daemon settings
+	DockerHostEnv = "DOCKER_HOST"
+	// DockerCertPathEnv is used for docker daemon settings
+	DockerCertPathEnv = "DOCKER_CERT_PATH"
+	// MinikubeActiveDockerdEnv holds the docker daemon which user's shell is pointing at
+	// value would be profile or empty if pointing to the user's host daemon.
+	// DockerDaemonEnvs has list of environment variables to control docker daemon shell is using
+
+	MinikubeActiveDockerdEnv = "MINIKUBE_ACTIVE_DOCKERD"
 )
 
+var DockerDaemonEnvs = [3]string{DockerHostEnv, DockerTLSVerifyEnv, DockerCertPathEnv}
```

Comment:
any reason to make this an array rather than a slice?

Reply:
Was gonna emulate a const. Emphasizing the length be fixed.